packages/api/README.md

```diff
@@ -1,19 +1,17 @@
-## Publicodes API
+## `@publicodes/api`
 
-Ce package vous permet de créer une API REST, qui suit les spécifications OpenAPI 3.0, à partir de vos règles Publicodes.
+This package allows you to create a REST API, which follows OpenAPI 3.0 specifications, from your Publicodes rules.
 
-Il consiste en un middleware [Koa](https://github.com/koajs/koa) qui ajoute des routes à votre router :
+It consists of [Koa](https://github.com/koajs/koa) middleware that adds routes to your router:
 
--   `POST` `/evaluate` Évalue une ou plusieurs expressions avec une situation donnée
--   `GET` `/rules` Retourne la liste de toutes vos règles
--   `GET` `/rules/{rule}` Retourne une règle spécifique
+-   `POST` `/evaluate` Evaluate one or more expressions with a given situation
+-   `GET` `/rules` Returns the list of all your rules
+-   `GET` `/rules/{rule}` Returns a specific rule
 
 ## Installation
 
 ```bash
 npm install @publicodes/api
-# or
-yarn add @publicodes/api
```

Comment:
FYI I removed yarn because:
1. people using `yarn` are familiar with package installation, the info in the README is only for absolute beginner in the JS ecosystem (we don't want to expose multiple install options)
2. there are other popular package managers such as `pnpm`, `bun`, etc.